src/main/java/org/micks/champmaker/auth/UserFilePersistService.java

```diff
@@ -20,4 +24,25 @@ public void storeUser(String username, String encryptedPassword) {
             throw new RuntimeException(e);
         }
     }
+
+    private User convertStr(String userLine) {
```

Comment:
Let's rename the method name to something more meaningful.

Reply:
fixed